### DISCUSSION TITLE
dubbo3如何在一个端口下同时支持http1.1和http2

### DISCUSSION THREAD

**Discussion by Webster-Yang** [apache/dubbo#14916] [Question] [answered]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java Samples (apache/dubbo-samples)

### Details

dubbo3如何在一个端口下同时支持http1.1和http2

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by oxsean** [comment, marked as answer]
It's supported by default.
